packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -117,7 +117,7 @@
 
 				<n8n-menu-item index="settings" v-if="canUserAccessSettings && currentUser">
 					<font-awesome-icon icon="cog"/>&nbsp;
-					<span slot="title" class="item-title-root">Settings</span>
+					<span slot="title" class="item-title-root">{{ $locale.baseText('SETTINGS') }}</span>
```

Comment:
```suggestion
					<span slot="title" class="item-title-root">{{ $locale.baseText('SETTINGS_MAIN_SIDEBAR_ITEM') }}</span>
```

Same for others where this happens.

Reply:
no because then we cannot reuse it.. the whole point of flattening is that I can reuse it.